source/release-notes/v1.3-release-notes.rst

```diff
@@ -65,7 +64,7 @@ Upgrading from v1.2
       haven't been doing this before then now is a good time to start. Feel
       free to contact us if you have trouble with this.
 
-   .. code-block:: shell
+   .. code-block:: sh
 
       # Copy over our app configuration (if they exist)
```

Comment:
Instead of a comment at the top, this should be a separate line of text outside of the code block.